tests/components/myuplink/test_switch.py

```diff
@@ -12,9 +14,7 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import entity_registry as er
 
-from . import setup_platform
-
-from tests.common import MockConfigEntry
+pytestmark = pytest.mark.parametrize("platforms", [(PLATFORM_DOMAIN,)])
```

Comment:
We use the `Platform` enum to populate platforms.

https://github.com/home-assistant/core/blob/16653ff5d00749ecb843f7d29cb031b083bf1c7e/homeassistant/const.py#L31